src/MspFcComms.cpp

```diff
@@ -60,6 +60,16 @@ namespace FcComms
     }
 
     // Send the rc commands to the FC using the member array of rc values.
+    FcCommsReturns MspFcComms::getRawRC()
+    {
+        MSP_RC msp_getrawRC;
+        msp_getrawRC.getRc(rc_values); // rc_values array initialized in corresponding .hpp file
+        #pragma GCC warning "Handle return"
+        sendMessage(msp_getrawRC);
+        ROS_INFO("rc_values are %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d", rc_values[0],  rc_values[1],  rc_values[2],  rc_values[3],  rc_values[4],
```

Comment:
It would be better to find a way to pack these values into a string and then pass that to ROS_INFO